packages/cli/src/daemon/__tests__/sse-feed.test.ts

```diff
@@ -1,86 +1,108 @@
-import { test, jest } from '@jest/globals'
-import type { SpiedFunction } from 'jest-mock'
+import { test, jest, expect } from '@jest/globals'
 import type { Response } from 'express'
-import { from, Observable } from 'rxjs'
-import { SseFeed } from '../sse-feed.js'
+import { ExpectedCloseError, SSESink } from '../sse-feed.js'
 import { EventEmitter } from 'node:events'
-import { LoggerProvider } from '@ceramicnetwork/common'
-
-const logger = new LoggerProvider().getDiagnosticsLogger()
+import { WritableStream } from 'node:stream/web'
 
 class FauxResponse extends EventEmitter {
   readonly writeHead = jest.fn()
-  readonly write = jest.fn()
+  readonly write = jest.fn(() => true)
   readonly end = jest.fn()
 }
 
-test('constructor does not subscribe to observable', () => {
-  const observable = from([1, 2, 3])
-  const subscribeSpy = jest.spyOn(observable, 'subscribe')
-  new SseFeed(logger, observable, String)
-  expect(subscribeSpy).not.toBeCalled()
-})
+function readableStreamFromIterable<T>(source: Iterable<T>) {
+  const iterator = source[Symbol.iterator]()
+  return new ReadableStream({
+    start(controller) {
+      const next = iterator.next()
+      if (next.done) {
+        controller.close()
+      } else {
+        controller.enqueue(next.value)
+      }
+    },
+    pull(controller) {
+      const next = iterator.next()
+      if (next.done) {
+        controller.close()
+      } else {
+        controller.enqueue(next.value)
+      }
+    },
+  })
+}
 
-test('send events', () => {
+test('send events 2', async () => {
```

Comment:
Why did we add the `2` in the test name?

Reply:
Whoops. Fixed. Remnant of dev time.